composer.json

```diff
@@ -20,7 +20,7 @@
   "require-dev": {
     "phpstan/phpstan": "^1.10.5",
     "phpstan/phpstan-symfony": "^1.2.20",
-    "gotenberg/gotenberg-php": "^1.0 || ^2.0",
+    "gotenberg/gotenberg-php": "^1.1.8 || ^2.2",
```

Comment:
@kingjia90 should I revert this change in this case? I guess versions `2.0` and `2.1` are also acceptable, but the metadata won't work.

Reply:
Thank you, mhhh, i personally would keep it, as long as it's documented in the minor release changelog, we should encourage to use a more recent dependency, and it would be simpler and easier for devs to keep in mind that the `metadata()` feature is not available ONLY to `^1.x`

Reply:
Makes sense 👍, resolving